docs/SwitchingToCodeCAPI.md

```diff
@@ -5,11 +5,11 @@ By convention, the Fronts CODE environment is pointed to PROD CAPI. This is to e
 Sometimes we need to be able to use content from CODE CAPI in the Fronts tool and Preview tool. To do this, we need to make two changes:
 
 ### Point `facia-tool` to CODE CAPI
-
-- Download the `facia-tool` configuration file from `s3://facia-private`.
-- Modify the `content.api.host`, `content.api.draft.iam-host` and `content.api.draft.role` settings in this file to the appropriate values. The Content Platform team should be able to provide these.
-- In AWS, update the `facia-tool` CODE stack (`facia-CODE` at the time of writing), changing the `CapiPreviewRole` to match the new value of `content.api.draft.role`. This will allow the `facia-tool` boxes to communicate with the new endpoints you've just changed in the configuration.
-- Redeploy the tool, which will refresh the configuration. You should see the feed of articles on the left hand side is now drawn from CAPI CODE.
+- Go to the AWS CMS Fronts console (can be accessed from Janus)
+- Download the `application secrets config` file from `s3://facia-private`.
```

Comment:
I would condense these points into one line: 

- Download the application secrets config from `s3://facia-private` in the CMS Fronts account.

We shouldn't need to point developers to the console (they can execute these steps from the command line if they want to).